tests/acceptance/test_v2_measurement_service.py

```diff
@@ -19,21 +19,8 @@ def test___streaming_measurement_service___request_number_of_responses___receive
     num_responses: int, stub_v2: v2_measurement_service_pb2_grpc.MeasurementServiceStub
 ):
     """End to End Test validating streaming measurement returns the expected number of responses."""
```

Comment:
Test cases in given/when/then format don't need docstrings anymore. 

Did you copy this from `test_measurement_service.py`? I thought I deleted all of the test case docstrings in #253, but apparently I missed one.

Reply:
Yes I did. `test___measurement_service_v2___measure___returns_output` still has a docstring.